lib/SetAPI/sampleset/SampleSetInterface.py

```diff
@@ -129,20 +112,20 @@ def create_sample_set(self, ctx, params):
                     ],
                 }
             )[0]
-            """
-            out_obj['sample_ids'] = [self._ws_get_obj_name(sample_id) for
-                                     sample_id in params['sample_ids']]
-            """
-            result = {}
-            result["set_ref"] = "{0}/{1}/{2}".format(res[6], res[0], res[4])
-            result["set_info"] = res
 
-            pprint(result)
-            return result
+            pprint(
```

Comment:
Does this need to be printed to the log? I guess it's probably useful for debugging.